### PR TITLE
Mutetoxin buff

### PR DESCRIPTION
**Pull request by Thatonestomf**
## About the PR

Made Mute toxin still mute you even after its metabolized, similar to glue
## Why / Balance

Mute toxin is heavily underpowered for its cost, and its expensive and still worse then glue at muting targets
## Technical details

i just added mute time to mute toxin
## Media

none needed
## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None i hope
**Changelog**

:cl:
- tweak: Mute toxin now mutes even after is metabolized, similar to glue
- tweak: Mute toxin is now even more powerful at muting people

### CODE REVIEW

**Comment by slarticodefast** [comment]
Looks like you branched off from the wrong branch. Your smile raffle change is in here.

**Comment by TurboTrackerss14** [comment]
It should add way more than 5s. 
Considering glue does the exact same timer, and is used to make mutetoxin.

**Comment by slarticodefast** [comment]
Or keep both at 5s and make glue not accumulate.

**Comment by Thatonestomf** [comment]
I wanted to avoid nerfing glue so ill just go ahead and buff Mute even more, although it may be a bit unfun to get hit by

**Comment by Thatonestomf** [comment]
alright i buffed it to 10 seconds of mute per tick and....
image
image

its a bit busted, but imma be real mute toxin is still too expensive, it may need a new recipe

**Comment by slarticodefast** [comment]
I guess it has to be pretty powerful to justify its cost, even if it means your target will have to play as a mime for a while 😆

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 1]
```diff
@@ -777,6 +777,7 @@
     name: ghost-role-information-smile-name
     description: ghost-role-information-smile-description
     rules: ghost-role-information-nonantagonist-rules
+    raffle: null
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:780]:
```suggestion
```
this is from another PR

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml (hunk 1 above)
Comment (Thatonestomf) [reply Resources/Prototypes/Entities/Mobs/NPCs/pets.yml]: thanks for getting that

**Review by slarticodefast — APPROVED** [review]
Tested it and while this is quite strong it feels reasonable considering how expensive mute toxin is to make (needing vestine and uranium). Could you adjust the changlog to include the strength increase?

**Comment by slarticodefast** [comment]
Fixes #32910

**Comment by Thatonestomf** [comment]
change log updated, 2 PRs done in 3 days, im feeling good

**Comment by Coolsurf6** [comment]
Even with this buff, I think it might be justified and could even be made stronger in the future. When things like PDA Messaging becomes a thing, people could just message about what occured using their PDA. I think this being an upgrade for glue seems justified imo.

**Comment by chromiumboy** [comment]
Let's give it a shot. If it ends up being too oppressive, it's not hard to change/revert

**Pull request merged**